### PR TITLE
Now track inventory when spinning a fort

### PR DESCRIPTION
**Pull request by BriceSD**
Now track inventory when spinning a fort

### CODE REVIEW

File: pokemongo_bot/cell_workers/spin_fort.py [hunk 1]
```diff
@@ -2,10 +2,12 @@
 from __future__ import unicode_literals
 
 import json
+import json
```
Comment (douglascamata) [inline pokemongo_bot/cell_workers/spin_fort.py]: duplicated import detected :P

**Comment by douglascamata** [comment]
👍 

LGTM. @TheSavior, what you think?

File: pokemongo_bot/cell_workers/spin_fort.py [hunk 2]
```diff
@@ -50,25 +57,16 @@ def work(self):
...
             player_longitude=f2i(self.bot.position[1])
         )
-        if 'responses' in response_dict and \
-                'FORT_SEARCH' in response_dict['responses']:
+        if 'responses' in response_dict and 'FORT_SEARCH' in response_dict['responses']:
 
             spin_details = response_dict['responses']['FORT_SEARCH']
             spin_result = spin_details.get('result', -1)
-            if spin_result == 1:
+            if spin_result == SPIN_REQUEST_RESULT_SUCCESS:
```
Comment (TheSavior) [inline pokemongo_bot/cell_workers/spin_fort.py]: Yesss!

File: pokemongo_bot/inventory.py [hunk 3]
```diff
@@ -106,6 +106,25 @@ def captured(self, pokemon_id):
             return False
         return self._data[pokemon_id]['times_captured'] > 0
 
+class Item(object):
```
Comment (TheSavior) [inline pokemongo_bot/inventory.py]: Didn't this just get moved below in a previous PR?

**Comment by TheSavior** [comment]
LGTM

:+1:

Merge conflicts though...

File: pokemongo_bot/inventory.py (hunk 3 above)
Comment (BriceSD) [inline pokemongo_bot/inventory.py]: Yes someone did. Maybe for the best. But when merging conflicts I simply accepted my modified version. We need to heavily refactor this class anyway, so I didn’t take the time to move it again.

File: pokemongo_bot/cell_workers/spin_fort.py (hunk 2 above)
Comment (BriceSD) [inline pokemongo_bot/cell_workers/spin_fort.py]: We should make a naming convention for these imo. I don’t like mine so if you got an idea

**Comment by BriceSD** [comment]
@douglascamata @TheSavior I resolved the confilcts

**Pull request merged**